plugins/coatils.py

```diff
@@ -15,9 +15,6 @@ class Coatils(BotPlugin):
     Various coala related utilities, exposing the REST API, etc.
     """
 
-    def __init__(self, bot, name=None):
```

Comment:
Unrelated change?

Reply:
Remove this change from the file and do `git commit --amend` to modify the commit(do not use git commit)